pkg/cluster/context.go

```diff
@@ -44,6 +44,7 @@ import (
 type Context struct {
 	name   string
 	status *logutil.Status
+	retain bool
```

Comment:
maybe just export this field for now, let create command set it, and I'll refactor in a follow-up. `NewContext` needs refactoring anyhow (going to delay validation..)